src/frontend/src/pages/PIAIntakeForm/index.tsx

```diff
@@ -492,10 +492,8 @@ const PIAFormPage = () => {
         )}
 
         <div className="component__container">
-          <section className="side-nav__container">
-            <PIASideNav pia={pia} isNewForm={!id}></PIASideNav>
-          </section>
-          <section className="form__container ms-md-auto content__container">
+          <PIASideNav pia={pia} isNewForm={!id}></PIASideNav>
+          <section className="ppq-form-section form__container ms-md-auto content__container">
```

Comment:
Please remove side modifications in your tab. These are constantly changing, and are handled separately.